src/Common/Key_Value_Cache/Table/Schema.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Defines the schema for the key-value cache table.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\Key_Value_Cache\Table;
+ */
+
+namespace TEC\Common\Key_Value_Cache\Table;
+
+use TEC\Common\StellarWP\Schema\Tables\Contracts\Table;
+
+/**
+ * Class Schema.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\Key_Value_Cache\Table;
+ */
+class Schema extends Table {
```

Comment:
You can skip this, but we also provide an internal table contract at `TEC\Common\Integrations\Custom_Table_Abstract`.

For which you can provide a definition of columns and all the select/insert/update/delete queries would be provided to you by the `Custom_Table_Query_Methods` trait.